testing/tests/DevExpress.ui.widgets/buttonGroup.tests.js

```diff
@@ -81,6 +82,17 @@ QUnit.module("option changed", {
         assert.ok(buttons.eq(1).hasClass(BUTTON_GROUP_ITEM_HAS_WIDTH), "second item when button group has width");
     });
 
+    QUnit.test("change the height option", function(assert) {
```

Comment:
Check another cases:

1) Init widget with a specific height
```js
  dxButtonGroup({
    height: 500
  });
```

2) Change height to `null`

```js
  dxButtonGroup({
    height: 500
  });
  <...>
  buttonGroup.option("height", null)
  // $(`.${BUTTON_GROUP_ITEM_HAS_HEIGHT}`).length => 0 
```